src/app/shared/navbar/navbar.scss

```diff
@@ -9,3 +11,7 @@
   height: 26px;
   vertical-align: middle;
 }
+
+.flex-spacer {
```

Comment:
It's mainly so that people who inspect the source of the docs site see which styles come from the docs and which are from material itself.